### DISCUSSION TITLE
关于对创造物品栏分栏的建议

### DISCUSSION THREAD

**Discussion by AoJiaoProuder** [IndustrialCraftMC/IndustrialCraft2#89] [【意见建议】Ideas]
例如AE2与TE等内容较多的物品会对创造物品栏多分几栏，比如把流体装饰方块矿石等放在一栏，工具武器装备等一栏，机器单独一栏。或者像植物魔法那样在分栏加入一个搜索栏，但因为加入这个以后没法在当前页面按e直接退出背包，所以在我们的小圈子中这个方法明显不如多分几栏受欢迎

**Comment by AbbottMc** [comment]
关于新增分栏这点我有考虑过，应该会在内容完善后进行重新设计
